contrib/python-bindings/source/triangulation_wrapper.cc

```diff
@@ -35,6 +35,43 @@ namespace python
 {
   namespace internal
   {
+    template <int dim, int spacedim>
+    void
+    create_triangulation(boost::python::list &vertices_list,
+                         boost::python::list &cells_vertices,
+                         void *               triangulation)
+    {
+      Triangulation<dim, spacedim> *tria =
+        static_cast<Triangulation<dim, spacedim> *>(triangulation);
+      tria->clear();
+
+      const size_t n_vertices = boost::python::len(vertices_list);
+      std::vector<Point<spacedim>> vertices(n_vertices);
+      for (size_t i = 0; i < n_vertices; ++i)
+        {
+          boost::python::list vertex =
+            boost::python::extract<boost::python::list>(vertices_list[i]);
+          for (int d = 0; d < spacedim; ++d)
+            vertices[i][d] = boost::python::extract<double>(vertex[d]);
+        }
+
+      const size_t               n_cells = boost::python::len(cells_vertices);
```

Comment:
Do you know why there is so many blank spaces? This looks strange.

Reply:
No idea. `make indent` insists on this...

Reply:
`clang-format` (with our settings) aligns subsequent variable declarations both at the type name and the variable name.